### PR TITLE
Features/nuget tasks

### PR DESCRIPTION
**Pull request by emanuelquintero**

### CODE REVIEW

File: Tasks/Common/nuget-task-common/package.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
   "license": "MIT",
   "dependencies": {
     "vso-node-api": "^5.0.5",
-    "vsts-task-lib": "^0.9.20"
+    "vsts-task-lib": "^0.9.20",
```
Comment (pspill) [inline Tasks/Common/nuget-task-common/package.json]: should update to v2

File: Tasks/NuGet/package.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "url": "git+https://github.com/Microsoft/vsts-tasks.git"
+  },
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Microsoft/vsts-tasks/issues"
+  },
+  "homepage": "https://github.com/Microsoft/vsts-tasks#readme",
+  "dependencies": {
+    "vsts-task-lib": "^0.9.20"
```
Comment (pspill) [inline Tasks/NuGet/package.json]: v2

File: Tasks/NuGet/task.json [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    "name": "NuGet",
+    "friendlyName": "NuGet Command",
+    "description": "Runs a NuGet command",
+    "category": "Package",
+    "author": "Microsoft Corporation",
+    "preview": "true",
+    "version": {
+        "Major": 0,
+        "Minor": 1,
+        "Patch": 2
```
Comment (pspill) [inline Tasks/NuGet/task.json]: 0

File: Tasks/NuGetToolInstaller/task.json [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    "runsOn": [
+        "Agent",
+        "DeploymentGroup"
+    ],
+    "minimumAgentVersion": "2.115.0",
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 0,
+        "Minor": 1,
+        "Patch": 3
```
Comment (pspill) [inline Tasks/NuGetToolInstaller/task.json]: 0

File: Tasks/NuGetInstaller/task.loc.json [hunk 5]
```diff
@@ -1,21 +1,21 @@
...
   "helpMarkDown": "ms-resource:loc.helpMarkDown",
   "category": "Package",
   "author": "Microsoft Corporation",
   "version": {
-    "Major": 0,
-    "Minor": 2,
-    "Patch": 31
+    "Major": 1,
+    "Minor": 0,
+    "Patch": 2
```
Comment (pspill) [inline Tasks/NuGetInstaller/task.loc.json]: 0

File: Tasks/NuGetInstaller/task.json [hunk 6]
```diff
@@ -1,79 +1,103 @@
...
     "helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?LinkID=613747)",
     "category": "Package",
     "author": "Microsoft Corporation",
     "version": {
-        "Major": 0,
-        "Minor": 2,
-        "Patch": 31
+        "Major": 1,
+        "Minor": 0,
+        "Patch": 2
```
Comment (pspill) [inline Tasks/NuGetInstaller/task.json]: 0

File: Tasks/NuGetInstaller/package.json [hunk 7]
```diff
@@ -18,6 +18,7 @@
   "homepage": "https://github.com/Microsoft/vsts-tasks#readme",
   "dependencies": {
     "vsts-task-lib": "^0.9.20",
```
Comment (pspill) [inline Tasks/NuGetInstaller/package.json]: v2

**Comment by bryanmacfarlane** [comment]
Let us know if you need any help getting it into 116 release branch.  Contact Eric or I

**Pull request merged**